lib/mongorito.js

```diff
@@ -866,9 +888,9 @@ exports = module.exports = Mongorito;
 
 exports.Model = Model;
 
-Object.keys(mongoskin).forEach(function (key) {
+Object.keys(mongodb).forEach(function (key) {
   if (['connect', 'version', 'db'].indexOf(key) === -1) {
-    exports[key] = mongoskin[key];
+    exports[key] = mongodb[key];
```

Comment:
Good question, I need to think on this, not sure what would be useful to expose beyond things like ObjectId. Do you have properties from mongoskin/monk that your using in code now?

mongodb exposes a number of interfaces like MongoClient, Db,  Cursor, ObjectId, etc.